circle.yml

```diff
@@ -13,3 +13,5 @@ test:
     - nvm install 0.12.7
     - nvm alias default 0.12.7
     - make test
+    - nvm use 4.2 && make test
+    - nvm use 5 && make test
```

Comment:
That makes sense. Is `5` equivalent to `5.0` or does it mean _the latest 5.x.x version_?